packages/cdktf-cli/bin/cmds/helper/init.ts

```diff
@@ -18,7 +18,7 @@ import { sendTelemetry } from "../../../lib/checkpoint";
 
 const chalkColour = new chalk.Instance();
 
-const templatesDir = path.join(__dirname, "..", "..", "..", "templates");
+const templatesDir = path.join(__dirname, "..", "templates");
```

Comment:
Will this cause issues when running the non-bundled version during local testing?

Reply:
If you mean actual usage of the locally build cli, then this shouldn't cause issues, since it's gonna bundle on `postbuild` (takes just one or two seconds)

Reply:
For actual test cases in terms of unit tests, this would likely cause issues